kin-ecosystem-sdk/src/main/java/com/kin/ecosystem/data/model/Coupon.java

```diff
@@ -5,15 +5,16 @@
 import com.google.gson.annotations.SerializedName;
 import com.google.gson.stream.JsonReader;
 import com.google.gson.stream.JsonWriter;
+import com.kin.ecosystem.network.model.CouponCodeResult;
```

Comment:
it is better not to use network/server object as it could change later, instead use and adapter object that provide the model the app needs